### PR TITLE
Introduce TopAppBar component

### PR DESCRIPTION
**Pull request by zurdus**

### CODE REVIEW

File: icons/src/main/java/kiwi/orbit/compose/icons/Icons.kt [hunk 1]
```diff
@@ -70,10 +70,18 @@ public object Icons {
         @Composable
         get() = painterResource(R.drawable.ic_orbit_apple)
 
+    public val ArrowBack: Painter
```
Comment (hrach) [inline icons/src/main/java/kiwi/orbit/compose/icons/Icons.kt:73]: This class is auto-generated, please use the new drawables directly with painterResource. 
I'd suggest also moving the resources to UI packages as we will avoid regenerating issues here.

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+import kiwi.orbit.compose.ui.OrbitTheme
+import kiwi.orbit.compose.ui.R
+import kiwi.orbit.compose.ui.foundation.ContentEmphasis
+import kiwi.orbit.compose.ui.foundation.LocalContentEmphasis
+import kiwi.orbit.compose.ui.foundation.ProvideMergedTextStyle
+import kiwi.orbit.compose.ui.foundation.contentColorFor
+import com.google.accompanist.insets.ui.TopAppBar as AccompanistTopAppBar
+
+@Composable
+public fun TopAppBar(
```
Comment (hrach) [inline ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:32]: We use other code style here - the official. It should work out of the box in IDE, doesn't it?

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+        )
+      }
+    },
+    actions = actions,
+  )
+}
+
+@Composable
+private fun getBackArrowIcon(): Painter {
+    val isRtl = LocalContext.current.resources.configuration.layoutDirection == View.LAYOUT_DIRECTION_RTL
```
Comment (hrach) [inline ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:112]: Use LocalLayoutDirection.current

File: icons/src/main/res/drawable/ic_orbit_arrow_back.xml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (hrach) [inline icons/src/main/res/drawable/ic_orbit_arrow_back.xml:1]: Please name these with left and right name -> the word "forward" (or backward") is RTL|LTR actually is dependent.

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+      {
+        CompositionLocalProvider(
+          LocalContentEmphasis provides ContentEmphasis.Normal
+        ) {
+          navigationIcon.invoke()
+        }
+      }
+    } else null,
+    contentPadding = rememberInsetsPaddingValues(
+      insets = LocalWindowInsets.current.statusBars,
```
Comment (hrach) [inline ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:74]: Please use systemBars to apply also padding for navigation bar.

File: ui/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="orbit_cd_back_button">Navigate up</string>
```
Comment (hrach) [inline ui/src/main/res/values/strings.xml:3]: this is Up, not back action, so the key should be named rather differently - `orbit_cd_navigate_up` ?

File: icons/src/main/res/drawable/ic_orbit_arrow_forward.xml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <group
+        android:name="rotationGroup"
+        android:pivotX="12"
+        android:scaleX="-1">
```
Comment (hrach) [inline icons/src/main/res/drawable/ic_orbit_arrow_forward.xml:10]: Nice trick!

**Review by hrach — CHANGES_REQUESTED** [review]

File: icons/src/main/res/drawable/ic_orbit_arrow_forward.xml (hunk 7 above)
Comment (zurdus) [reply icons/src/main/res/drawable/ic_orbit_arrow_forward.xml:10]: 😁 pretty handy, right?

File: icons/src/main/java/kiwi/orbit/compose/icons/Icons.kt (hunk 1 above)
Comment (zurdus) [reply icons/src/main/java/kiwi/orbit/compose/icons/Icons.kt:73]: Done.

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt (hunk 2 above)
Comment (zurdus) [reply ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:32]: Done.

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt (hunk 3 above)
Comment (zurdus) [reply ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:112]: I didn't know this one existed, thanks.

File: icons/src/main/res/drawable/ic_orbit_arrow_back.xml (hunk 4 above)
Comment (zurdus) [reply icons/src/main/res/drawable/ic_orbit_arrow_back.xml:1]: Done.

File: ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt (hunk 5 above)
Comment (zurdus) [reply ui/src/main/java/kiwi/orbit/compose/ui/controls/TopAppBar.kt:74]: Done.

File: ui/src/main/res/values/strings.xml (hunk 6 above)
Comment (zurdus) [reply ui/src/main/res/values/strings.xml:3]: Done.

**Review by hrach — APPROVED** [review]

**Pull request merged**